src/plugins/vue-i18n.js

```diff
@@ -4,8 +4,8 @@ export default () => {
     silentFallbackWarn: true,
     pluralizationRules: {
       /**
-       * @param choice {number} a choice index given by the input to $tc: `$tc('path.to.rule', choiceIndex)`
-       * @param choicesLength {number} an overall amount of available choices
+       * @param choice - {number} a choice index given by the input to $tc: `$tc('path.to.rule', choiceIndex)`
```

Comment:
Hm, intersting. It doesn't recognize that there is a type annotation that should be switched with the param name here, and simply adds a dash in front of it.

Reply:
It's because the order is wrong, the type annotation comes before the name in JSDoc :slightly_smiling_face: